public/api-docs/docs/request.yaml

```diff
@@ -0,0 +1,158 @@
+components:
```

Comment:
I Don't think all these swagger reconfiguration should be in this PR.

Reply:
It came from develop after the login features was merged. But I didn't add anything there.

Reply:
@UggoPrince   I believe these commits came from when you merge the swagger reconfiguraton branch with develop ealier. Please, kindly update your local develop branch with what we have currently on the remote develop branch.

Reply:
that branch is yet to be merged. But i will update with what we have now. When the branch gets merged then we continue with it.

Reply:
I was referring to this PR `#168249689 refactor codebase for swagger` that was erroneously merged previously.

Reply:
no it has not been merged. All that yaml is login feature branch i thinking the refactor swagger branch will be merged first. I've removed it though.